pkg/requirements.pip

```diff
@@ -1,5 +1,8 @@
 # if we bump the gnupg version, bump also the sanity check
 # in keymanager.__init__
+
+https://launchpad.net/dirspec/stable-13-10/13.10/+download/dirspec-13.10.tar.gz
 gnupg>=1.4.0
 simplejson
 requests
+leap.common>=0.4.3
```

Comment:
we don't want leap dependencies in the main requirements, they should go in requirements-leap. this causes problems when we bump dependencies during a release cycle and the required dependency has not yet been published.